src/app/core/campaign/campaign.service.spec.ts

```diff
@@ -49,6 +51,11 @@ describe('CampaignService', () => {
       valid: true,
       flights: { 9: flightStateFixture }
     };
+    augury = new MockHalService();
+    campaignDoc = augury.mock('prx:campaign', campaignFixture);
+    store = of({ campaignState: { ...campaignStateFixture, doc: campaignDoc } }) as any;
```

Comment:
Might want to create a TestBed for this spec at some point.
I didn't want to mock up the entire store setup here to include the campaign doc selector, so I mocked the store just as an Observable that returns state.